ui/src/Pages/Preferences/Playback/EnableSsa.tsx

```diff
@@ -1,14 +1,14 @@
 import { useState, useCallback, useEffect } from "react";
 
-import Toggle from "../../../Components/Toggle";
+import Toggle from "Components/Toggle";
 
 function EnableSsa() {
   const [enableSsa, setEnableSsa] = useState(
     localStorage.getItem("enable_ssa") === "true"
   );
 
   useEffect(() => {
-    localStorage.setItem("enable_ssa", enableSsa);
+    localStorage.setItem("enable_ssa", enableSsa.toString());
```

Comment:
Im wondering if enableSsa should be stored as a boolean, but I guess this is fine for now.

Reply:
All keys and values in local storage are strings. The `toString` is to satisfy the TS compiler, which expects the second argument to be a string.